gpt_pp/ai/__init__.py

```diff
@@ -89,15 +89,15 @@ def get_chat(self, files: str) -> ConversationChain:
             memory=memory,
```

Comment:
The updated spinner text in line 89 is more accurate for the task at hand. Nice improvement!